config/configgrpc/configgrpc_test.go

```diff
@@ -540,7 +540,7 @@ func TestGRPCClientSettingsError(t *testing.T) {
 	}
 	for _, test := range tests {
 		t.Run(test.err, func(t *testing.T) {
-			require.NoError(t, test.settings.Validate())
+			require.Error(t, test.settings.Validate())
```

Comment:
I don't see anything on why we should expect this to _not_ throw an error, so if I've missed something let me know and I'll document this behavior instead.